Source/Core/DolphinQt2/MainWindow.cpp

```diff
@@ -379,6 +408,14 @@ void MainWindow::ShowAboutDialog()
   about->show();
 }
 
+void MainWindow::ShowHotkeyDialog()
+{
+  m_hotkey_window->ChangeMappingType(MappingWindow::Type::MAPPING_HOTKEYS);
+  m_hotkey_window->show();
```

Comment:
Well, I do not intend to wait for the window. So ``show`` should be fine (Except if that's the part you're worried about)?